### PR TITLE
added language config

### PR DESCRIPTION
**Pull request by naveen-egov**
added languages and localisation modules

### CODE REVIEW

File: data/default/health/field-app-configuration.json [hunk 1]
```diff
@@ -9,5 +9,23 @@
     "SYNC_METHOD": "API | OPLOG_UPLOAD",
     "SYNC_TRIGGER": "MANUAL | NETWORK_AVAILABLE"
     }
+    
+  ],
+  "languages": [
```
Comment (egov-joy) [inline data/default/health/field-app-configuration.json]: Please move 'languages' and 'localizationModules' within AppConfig

File: data/default/health/field-app-configuration.json [hunk 2]
```diff
@@ -9,5 +9,23 @@
...
+    {
+       "label": "Portuguese",
+       "value": "pt_MZ"
+    },
+
+    {
+      "label": "ENGLISH",
```
Comment (egov-joy) [inline data/default/health/field-app-configuration.json]: Use 'English' instead of 'ENGLISH'

**Review by egov-joy — CHANGES_REQUESTED** [review]

File: data/default/health/field-app-configuration.json [hunk 3]
```diff
@@ -9,5 +9,23 @@
...
+      "value": "en_MZ"
+    }
+  ],
+  "localizationModules": [
+    {
+      "label": "hcm-common",
```
Comment (egov-joy) [inline data/default/health/field-app-configuration.json]: do we need label and value for the module names?

**Review by egov-joy — CHANGES_REQUESTED** [review]

File: data/default/health/field-app-configuration.json (hunk 3 above)
Comment (naveen-egov) [reply data/default/health/field-app-configuration.json]: yea,  i have added label and name by referring digit code base.

**Review by egov-joy — APPROVED** [review]

**Pull request merged**